variables.tf

```diff
@@ -18,6 +18,40 @@ variable "assign_generated_ipv6_cidr_block" {
   default     = false
```

Comment:
would it make sense to remove this variable and set this `assign_generated_ipv6_cidr_block = "${var.enable_ipv6}"`?

Reply:
Ah.. now I see that you actually mentioned this in the PR description..

Reply:
Yup, I didn't want to remove the option until I had a chance to hear how you wanted to handle it. I wanted to remove the `assign_generated_ipv6_cidr_block` variable entirely and just set have the new variable name be `enable_ipv6`. Alternatively, keeping the assign name and making it actually set up the ipv6 cidr associations. Do you have any thoughts on how you'd like it to be handled?